### DISCUSSION TITLE
Cesium文件请求失败

### DISCUSSION THREAD

**Discussion by ChauPha** [zouyaoji/vue-cesium#207] [Q&A]
请求cdn上的cesium资源时经常失败
![1652755669(1)](https://user-images.githubusercontent.com/24471468/168718238-eee3c1f5-1c85-41d0-b4ce-a43de7881280.jpg)

**Comment by ChauPha** [comment]
您好 今天加载vue-cesium一直提示cdn上的资源找不到 可以帮忙解答下吗@zouyaoji

**Reply by webdq** [reply]
网络问题

**Reply by ChauPha** [reply]
这不是我本地的网络问题吧

**Reply by zouyaoji** [reply]
在你那 jsdelivr.com 被墙了？

**Reply by ChauPha** [reply]
是的 一天了都这样 一直显示访问不了了

**Reply by adwii** [reply]
> 在你那 jsdelivr.com 被墙了？
我项目也出现这种问题了，cdn.jsdelivr.net请求报错ERR_CONNECTION_RESET

**Reply by zouyaoji** [reply]
我今天发一个版本，默认改成unpkg的。

**Comment by ChauPha** [comment]
这个不是我本地的网络问题吧&nbsp;



------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年5月17日(星期二) 下午5:00
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [zouyaoji/vue-cesium] Cesium文件请求失败 (Discussion #207)





 
网络问题
